src/components/common/buttons.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'lodash/fp'
```

Comment:
Alternatively, each button could have its own module. But given how simple and closely related these 3 button components are, I think grouping them together is fine.